source/manual/government-changes.html.md.erb

```diff
@@ -166,94 +166,14 @@ in Whitehall. One of these is perceived to be a more senior role than the other.
 However, [Collections](https://github.com/alphagov/collections) will just render
```

Comment:
I think `However, [Collections](https://github.com/alphagov/collections) will just render these by ascending order of ActiveRecord IDs as default.` can now be removed, since Collections actually renders these based on the order they are included in the content item (which is controlled by Whitehall).